release-notes.md

```diff
@@ -1,9 +1,86 @@
 ---
-layout: archive
-title: "Release Notes"
+layout: post
+title: Armory Release Notes
 permalink: /release-notes/
-description: Add a description here.
+description: Release Notes For Armory Spinnaker
 collection: release_notes
 ---
+* This is a placeholder for an unordered list that will be replaced with ToC. To exclude a header, add {:.no_toc} after it.
+{:toc}
 
-This is a placeholder to point at the custom archive.html layout.
+
+## Understanding Armory + Open Source Spinnaker releases
+Armory is based off of OSS's (Open Source Spinnaker) [Halyard release Cadence](https://www.spinnaker.io/community/releases/release-cadence), in which we extend OSS with Armory features. We provide a few different release types.
+
+A **stable** release for customers who need stability in their production environment. You can find these [releases here](#stable-armory-releases) or use [Armory Halyard](#differences-between-releases).
+
+For customers who want to test the latest Armory + OSS (Open Source Spinnaker) release, we provide **rc** release candidates, which you can find using [Armory Halyard](#differences-between-releases). See OSS's [Release Cadence](https://www.spinnaker.io/community/releases/release-cadence/) for more information.
+
+We also provide nightly **ossedge** builds from **OSS master**, which is only built, but not tested by Armory, which you can find using [Armory Halyard](#differences-between-releases).
```

Comment:
Break this into two sentences, shouldn't have two "which"s in the same sentence.  Basically put a period after "tested by Armory" and make the second sentence just "You can find these builds using Armory Halyard"